Source/Scene/Scene.js

```diff
@@ -2468,7 +2468,7 @@ define([
             scene._fxaa.clear(context, passState, clearColor);
         }
 
-        if (environmentState.isSunVisible && scene.sunBloom && !useWebVR) {
+        if (environmentState.isSunVisible && scene.sunBloom && !useWebVR && !picking) {
```

Comment:
Maybe the environment isn't being updated correctly?  The commit that caused this (a9a275f) changes the location of the environment update.